src/test/ui/lifetime-errors/ex1-return-one-existing-name-if-else-3.stderr

```diff
@@ -2,7 +2,7 @@ error[E0621]: explicit lifetime required in parameter type
   --> $DIR/ex1-return-one-existing-name-if-else-3.rs:12:27
    |
 11 | fn foo<'a>((x, y): (&'a i32, &i32)) -> &'a i32 {
-   |            ------ consider changing type to `(&'a i32, &'a i32)`
+   |                              ---- consider changing type to `(&'a i32, &'a i32)`
```

Comment:
Hmm, so here we are underlining only a piece of the type, but the replacement that we are suggesting is the *entire* type. This seems a bit confusing to me.